doc/source/serve/autoscaling-guide.md

```diff
@@ -0,0 +1,349 @@
+
+**look_back_period_s[default_value=30]**: This controls how often each replica sends reports on current ongoing requests to the autoscaler. Note that the autoscaler cannot make new decisions if it does not receive updated metrics, so you most likely want to set `metrics_interval_s` to a value that is at most the upscale and downscale delay values. For instance, if you set `upscale_delay_s = 3`, but keep `metrics_interval_s = 10`, the autoscaler will only upscale roughly every 10 seconds.
+
+
+## Troubleshooting Guide
+
+
+### Number of Autoscaled Replicas not Stable
+
+If you are seeing that the number of replicas in your deployment keeps oscillating even though the traffic is relatively stable, here are some things you can try:
+
+* Set a smaller `upscale_smoothing_factor` and `downscale_smoothing_factor`.
+* Set a `look_back_period_s` value that matches the rest of the autoscaling config. For longer upscale and downscale delay values, a longer look back period can likely help stabilize the replica graph, but for shorter upscale and downscale delay values, a shorter look back period may be more appropriate. For instance, the following replica graphs show how a deployment with `upscale_delay_s = 3` works with a longer vs shorter look back period.
+
+| `look_back_period_s = 30` | `look_back_period_s = 3` |
+| ------------------------------------------------ | ----------------------------------------------- |
+| ![look-back-before](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/look_back_period_before.png) | ![look-back-after](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/look_back_period_after.png) |
+
+
+### High spikes in latency during bursts of traffic
+
+If you expect your application to receive bursty traffic, and at the same time want the deployments to scale down in periods of inactivity, you are likely concerned about how quickly the deployment can scale up and respond to bursts of traffic. While an increase in latency initially during a burst in traffic may be unavoidable, here are some things you can try to improve latency during bursts of traffic.
+
+* Set a lower `upscale_delay_s`. The autoscaler will always wait `upscale_delay_s` seconds before making a decision to upscale, so lowering this delay allows the autoscaler to react more quickly to changes, especially bursts, of traffic.
+* Set a larger `upscale_smoothing_factor`. If `upscale_smoothing_factor > 1`, then the autoscaler will scale up more aggressively than normal. This can allow your deployment to be more sensitive to bursts of traffic.
+* Lower the `metric_interval_s`. we recommend to always set `metric_interval_s` to be less than or equal to `upscale_delay_s`, otherwise upscaling will be delayed because the autoscaler doesn't receive fresh information often enough.
```

Comment:
```suggestion
* Lower the `metric_interval_s`. Always set `metric_interval_s` to be less than or equal to `upscale_delay_s`, otherwise upscaling is delayed because the autoscaler doesn't receive fresh information often enough.
```